api/src/server/db/repository/game/character.repository.ts

```diff
@@ -7,17 +7,18 @@ import {
 import { RaceKeyProvider } from '@/domain/game/tools/character-race-provider.tool';
 import { IdentifierProviderService } from '@/domain/shared/identifier-provider.tool';
 
+import { connection } from '../..';
 import { Character as CharacterEntity } from '../../entities/game/character';
 
 export const createEntityCharacterRepository = <TRaceKey extends string>(
   raceKeyProvider: RaceKeyProvider<TRaceKey>,
   identifierProviderService: IdentifierProviderService
 ): CharacterRepository<TRaceKey> => {
-  const charRepository = getRepository(CharacterEntity);
+  const charRepository = connection.getRepository(CharacterEntity);
   const createCharacter = createNewCharacter<TRaceKey>(
     identifierProviderService
   );
-  const entityToModel = mapEntityToModel<TRaceKey>(raceKeyProvider);
+  const entityToModel = getEntityToModelMapper<TRaceKey>(raceKeyProvider);
```

Comment:
if we are creating a function within a function we should name it 

`getThisFunctionName` or `createThisFunctionName`

I'm asking about style/convention

Reply:
we do not have a convention for this. I think it will not be so common like it looks like, as we will introduce more classes in repositories and other places soon.
I just want to be more obvious what the function do: `getEntityToModelMapper` tell as that it provide a mapper.
old `mapEntityToModel` suggested that it map entity to model (but it did not, it was factory for such function).